test/src/main/configurations/HTTP/ConfigurationWebServiceListenerSenderAttachments.xml

```diff
@@ -170,18 +170,18 @@
 				/>
 				<forward name="success" path="WriteStream" />
 			</pipe>
-			<pipe
-				name="WriteStream" className="nl.nn.adapterframework.pipes.FilePipe"
-				actions="write" getInputFromSessionKey="multipart1" filename="${testdata.dir}/webservices/received.txt"
+			<LocalFileSystemPipe
+				name="WriteStream"
+				action="write" getInputFromSessionKey="multipart1" filename="${testdata.dir}/webservices/received.txt"
 			>
 				<forward name="success" path="ReadFile"/>
-			</pipe>
-			<pipe
-				name="ReadFile" className="nl.nn.adapterframework.pipes.FilePipe"
-				actions="read" filename="${testdata.dir}/webservices/received.txt"
+			</LocalFileSystemPipe>
+			<LocalFileSystemPipe
+				name="ReadFile"
+				action="read" filename="${testdata.dir}/webservices/received.txt"
```

Comment:
It appears to me that you write the message to a file and then read that file again. That does not test the repeatability of the message. You could test that by reading it twice, e.g. by writing it once, and then appending it to the same file. Then you end up with a file with 'double contents'

Reply:
This part did not change. This part is just partially getting rid of the console warnings. Filepipe to localfilesystempipe

Reply:
OK